File: CooCooBase/Resources/Custom UI/PurchaseTicketTableViewCell.m
//
//  PurchaseTicketTableViewCell.m
//  CooCooBase
//
//  Created by ibasemac3 on 12/14/17.
//  Copyright © 2017 CooCoo. All rights reserved.
//

#import "PurchaseTicketTableViewCell.h"
#import "CooCooBase.h"

@implementation PurchaseTicketTableViewCell

- (void)awakeFromNib {
    [super awakeFromNib];
    // Initialization code
    self.plusButton.backgroundColor = [UIColor colorWithHexString:[Utilities colorHexStringFromId:[Utilities buttonBGColor]]] ;
    self.minusButton.backgroundColor = [UIColor colorWithHexString:[Utilities colorHexStringFromId:[Utilities buttonBGColor]]] ;
}
-(void)prepareForReuse{
    [super prepareForReuse];
}
- (void)setSelected:(BOOL)selected animated:(BOOL)animated {
    [super setSelected:selected animated:animated];

    // Configure the view for the selected state
}

@end

File: Genfare/ViewControllers/Store/PurchaseTicketTableViewCell.swift
//
//  PurchaseTicketTableViewCell.swift
//  Genfare
//
//  Created by omniwyse on 30/01/19.
//  Copyright © 2019 Omniwyse. All rights reserved.
//

import UIKit

class PurchaseTicketTableViewCell: UITableViewCell {

    @IBOutlet var RiderName: UILabel!
    @IBOutlet var TicketAmount: UILabel!
    @IBOutlet var RiderTypeDesc: UILabel!
    @IBOutlet var FareZoneCodeDesc: UILabel!
    @IBOutlet var TicketCount: UILabel!
    @IBOutlet var TotalTicketFare: UILabel!
    @IBOutlet var PlusButton: GFMenuButton!
    
    @IBOutlet var MinusButton: GFMenuButton!
    
    func setProductData(product:Product)
    {
        self.RiderName.text = product.productDescription
        self.TotalTicketFare.text = product.ticketTypeDescription
        
    }
    override func awakeFromNib() {
        super.awakeFromNib()
        // Initialization code
    }

    override func setSelected(_ selected: Bool, animated: Bool) {
        super.setSelected(selected, animated: animated)

        // Configure the view for the selected state
    }

}
